src/map/battlefield_handler.cpp

```diff
@@ -36,183 +36,122 @@
 #include "recast_container.h"
 #include "status_effect_container.h"
 
-CBattlefieldHandler::CBattlefieldHandler(uint16 zoneid)
-{
+CBattlefieldHandler::CBattlefieldHandler(uint16 zoneid) {
     m_ZoneId = zoneid;
 
-    //Dynamis zone (need to add COP dyna zone)
-    //added ghelsba outpost here, 1 battlefield only
-    if (m_ZoneId > 184 && m_ZoneId < 189 || m_ZoneId > 133 && m_ZoneId < 136 || m_ZoneId == 140 || m_ZoneId == 35 || m_ZoneId > 38 && m_ZoneId < 43)
-    {
+    // The Garden of RuHmet, Dynamis zones, and Ghelsba Outpost
+    if (m_ZoneId == 35 || m_ZoneId > 38 && m_ZoneId < 43 || m_ZoneId > 133 && m_ZoneId < 136 || m_ZoneId == 140 || m_ZoneId > 184 && m_ZoneId < 189 ) {
```

Comment:
not sure we should keep these ids hardcoded (i hate them)
maybe something like https://github.com/DarkstarProject/darkstar/blob/bcnmshit/src/map/battlefield_handler.cpp#L57